### PR TITLE
Feature/koo edit index

### PR DESCRIPTION
**Pull request by blxdaniel**
This is the basic page all teachers will have
Weeeee, my first pull request

### CODE REVIEW

**Comment by blxdaniel** [comment]
Can I only assign one person to check the PR?

**Comment by dzmitry-tsurankou** [comment]
@blxdaniel yep, only one person

File: team-koo/css/style.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.length {
+	padding-bottom: 100%;
```
Comment (dzmitry-tsurankou) [inline team-koo/css/style.css]: indentation should be 2 spaces.

File: team-koo/index.html [hunk 2]
```diff
@@ -1,38 +1,74 @@
-<!DOCTYPE html>
+<!DOCTYPE <!DOCTYPE html>
```
Comment (dzmitry-tsurankou) [inline team-koo/index.html]: this line looks broken to me

File: team-koo/index.html [hunk 3]
```diff
@@ -1,38 +1,74 @@
...
+	</nav>
+	<section>
+		<div class="container-fluid">
+			<div class="row">
+				<div class="col-lg-3 col-md-3 col-sm-3 col-xs-12 length">
+						<ul class="nav nav-pills nav-stacked">
+							<li role="presentation"><a href="#">Home</a></li>
+							<li role="presentation" class="active"><a href="#">Teachers</a></li>
+							<li role="presentation"><a href="#">Contact Us</a></li>
+							<!Here @ col-xs-12, I want a paragraph at the bottom to display>
```
Comment (dzmitry-tsurankou) [inline team-koo/index.html]: Is it supposed to be commented out?

**Comment by blxdaniel** [comment]
The comment is suppose to be there so I can get back to it. I needed to give my members a page layout that is set (That small portion is something I need to figure out)

**Comment by blxdaniel** [comment]
@dzmitry-tsurankou alright, everything should be fixed and ready to go!

**Comment by dzmitry-tsurankou** [comment]
it's not a comment. the proper way of commenting in html is
```<!-- comment text -->```

File: team-koo/index.html [hunk 4]
```diff
@@ -1,38 +1,74 @@
...
-		<div id="banner">
-			<section>
-				<ul>
-					<li>
-						<a href="img/sunset.jpg"></a>
-						<img src="img/sunset.jpg" alt="">
-						<p id = "para">We will have a banner with dvc teachers</p>
-					</li>
+<head>
+	<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
```
Comment (dzmitry-tsurankou) [inline team-koo/index.html]: and indent 2 spaces here too, please.

File: team-koo/index.html [hunk 5]
```diff
@@ -1,38 +1,74 @@
...
-				<p> &copy; 2015 Daniel Koo</p>
-			</footer>	
+				<form class="navbar-form navbar-left" role="search">
+					<div class="form-group">
+						<input type="text" class="form-control" placeholder="Search">
+					</div>
+					<button type="submit" class="btn btn-default">Submit</button>
+				</form>
+			</div><!-- /.navbar-collapse -->
+		</div><!-- /.container-fluid -->
```
Comment (dzmitry-tsurankou) [inline team-koo/index.html]: here is an example of how to comment

**Comment by blxdaniel** [comment]
@dzmitry-tsurankou So, both on sumblime and the commit i put in 2 space indentation, yet when I save the commit, it bumps it back to 4 spaces?

**Comment by dzmitry-tsurankou** [comment]
@blxdaniel It should not happen. you need to go to specify indentation in sublime for every file, that might be the problem. or you are just editing the wrong file.

File: team-koo/css/style.css [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  margin-right: 10px;
+
+}
+.textArea{
+  margin: 0 auto;
+}
+
+.fit-image{
+  max-width: 80px;
+  max-height:30px;
```
Comment (dzmitry-tsurankou) [inline team-koo/css/style.css]: please, put a space after ```:``` here

File: team-koo/css/style.css [hunk 7]
```diff
@@ -0,0 +1,18 @@
+.length {
+  padding-bottom: 100%;
+  background-color: white;
+}
```
Comment (dzmitry-tsurankou) [inline team-koo/css/style.css]: please, put a space after this line

File: team-koo/css/style.css [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+.picture {
+  margin-top: 5px;
```
Comment (dzmitry-tsurankou) [inline team-koo/css/style.css]: you can use ```margin: 5px 10px 10px 0``` instead. That will save you some lines

**Comment by blxdaniel** [comment]
@dzmitry-tsurankou lol... so many problems :crying_cat_face:

File: team-koo/index.html [hunk 9]
```diff
@@ -1,38 +1,74 @@
...
-					<li>
-						<a href="img/sunset.jpg"></a>
-						<img src="img/sunset.jpg" alt="">
-						<p id = "para">We will have a banner with dvc teachers</p>
-					</li>
+<head>
+	<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<title>ICanPass | Dr.Burns</title>
+</head>
```
Comment (dzmitry-tsurankou) [inline team-koo/index.html]: please, indent ```head``` and ```body``` and everything inside one level up, since it those are inside of ```<html> ```

**Comment by dzmitry-tsurankou** [comment]
Sorry for being an ass :) Those are not really problems, just style corrections. No worries, we'll get it merged.

**Comment by blxdaniel** [comment]
@dzmitry-tsurankou  HAHAHA nah, it's good. It's all in good practice so that I don't do this when this is before a career lol

**Comment by blxdaniel** [comment]
@dzmitry-tsurankou whew. Okay, yesyesyes?

**Comment by dzmitry-tsurankou** [comment]
you earned it :shipit:

**Pull request merged**